src/redisraft.c

```diff
@@ -156,29 +162,54 @@ static int cmdRaftTransferLeader(RedisModuleCtx *ctx, RedisModuleString **argv,
         return REDISMODULE_OK;
     }
 
-    RaftReq *req = RaftReqInit(ctx, RR_TRANSFER_LEADER);
-    req->r.node_to_transfer_leader = target_node_id;
+    int err = raft_transfer_leader(rr->raft, target_node_id, 0);
+    if (err != 0) {
+        switch (err) {
+            case RAFT_ERR_NOT_LEADER:
```

Comment:
Probably a good idea to find a way to consolidate this with `replyRaftError()`.

Reply:
Okay, moved these error cases into `replyRaftError()`